src/Nancy/Extensions/StreamExtensions.cs

```diff
@@ -5,17 +5,17 @@
     using Nancy.IO;
 
     /// <summary>
-    /// Extensions for RequestStream.
+    /// Extensions for Stream.
     /// </summary>
-    public static class RequestStreamExtensions
+    public static class StreamExtensions
     {
         /// <summary>
         /// Gets the request body as a string.
         /// </summary>
-        /// <param name="stream">The request body stream.</param>
+        /// <param name="stream">Stream.</param>

```

Comment:
Yes 👍 , I wasn't sure if I should change that. Same with "Gets the request body as a string."